button-examples/Android/android-button-example.md

```diff
@@ -215,9 +208,9 @@ An outlined button has a text label, a stroked container and an optional icon.
 
 ![Outlined button anatomy diagram](assets/outlined_button_anatomy.png)
 
-1. Text label
-1. Container
-1. Icon
+* A. Text label
```

Comment:
We're going to eventually replace the letter callouts with numbers.

Reply:
got it! will revert it.

Reply:
FYI, anatomy diagrams updated in my branch.